test/jobs/search_deindexer_job_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+class MockDeindexerModel
```

Comment:
I like the idea of just declaring these mocks inline. Reading these tests is slowly helping me get over withdrawal from RSpec